evap/static/css/evap.css

```diff
@@ -511,10 +511,19 @@ li.menu-indent {
 }
 
 .text-answer li,a {
-    word-break: break-word;
+    word-wrap: break-word;
```

Comment:
Ok, i thought that comment was referring to the possible origin of the original error.